mbedtls/src/pk/mod.rs

```diff
@@ -1543,7 +1543,7 @@ iy6KC991zzvaWY/Ys+q/84Afqa+0qJKQnPuy/7F5GkVdQA/lfbhi
         });
         assert_eq!(
             pk.verify(digest, data, &signature[0..len]).unwrap_err(),
-            Error::RsaInvalidPadding
+            codes::RsaInvalidPadding.into()
         );
     }
 
```

Comment:
The error types in tests are not updated correctly according to the failed CI. Please update them

Reply:
addressed this comment.